prime-router/src/main/kotlin/ConsolidatedActionLog.kt

```diff
@@ -0,0 +1,78 @@
+package gov.cdc.prime.router
+
+import com.fasterxml.jackson.annotation.JsonIgnore
+import com.fasterxml.jackson.annotation.JsonInclude
+import com.fasterxml.jackson.annotation.JsonInclude.Include
+
+/**
+ * Consolidated action log class to be output to the API JSON response.
+ * @param log the base log message to be consolidated
+ */
+@JsonInclude(Include.NON_NULL)
+class ConsolidatedActionLog(log: DetailActionLog) {
```

Comment:
You have one file per class, which is more like a Java or C type of thing.  Kotlin allows to bundle more classes into one file.  Bundle some of these that are related into one file to limit small files.